src/geo/ui/infowindow-view.js

```diff
@@ -107,8 +111,12 @@ var Infowindow = View.extend({
 
       var data = this.model.get('content') ? this.model.get('content').data : {};
 
-      // Sanitized fields
-      fields = _.map(fields, this._sanitizeField, this);
+      // If a custom template is not applied, let's sanitize
+      // fields for the template rendering
+      if (!this.isCustomTemplate()) {
```

Comment:
Aren't all templates "custom" in the context of cartodb.js now that we don't rely on `template_name` anymore?